snuba/redis.py

```diff
@@ -137,6 +138,9 @@ class RedisClientKey(Enum):
     RedisClientKey.OPTIMIZE: _initialize_specialized_redis_cluster(
         settings.REDIS_CLUSTERS["optimize"]
     ),
+    RedisClientKey.ADMIN_AUTH: _initialize_specialized_redis_cluster(
+        settings.REDIS_CLUSTERS["admin_auth"]
```

Comment:
Is there a PR to add `admin_auth` to the prod settings as well?

Reply:
not yet, I'll make a PR right now

Reply:
That will need to go out before this can be merged.

Reply:
this stuff is super inconvenient. I wonder if we can make it so that the prod_settings file can override individual keys of the default settings file, and does not have to redefine the entire variable